apps/datadogmonitors/datadog_monitors.star

```diff
@@ -0,0 +1,138 @@
+""")
+
+CACHE_KEY_PREFIX = "monitors_cached"
+DEFAULT_QUERY = "status:alert"
+
+def main(config):
```

Comment:
The API key and app key will likely be empty when we render previews in the Tidbyt mobile app. Two things you should consider:
- [blocker] Make sure you're not caching on empty keys
- [nice to have] Add a default view so that we have a nice preview in the mobile app

Reply:
Ah great callouts thank you! I will correct those later this afternoon!